src/index.html

```diff
@@ -4,9 +4,283 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header">
+      <a href="#" class="header__logo">
+        <img src="../src/images/logo.png" alt="#">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link nav__link_is-active">APPLE</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">SAMSUNG</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">СМАРТФОНЫ&nbsp;И&nbsp;ТЕЛЕФОНЫ</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              НОУТБУКИ&nbsp;И&nbsp;КОМПЬЮТЕРЫ
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">ГАДЖЕТЫ</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">ПЛАНШЕТЫ</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">ФОТО</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">ВИДЕО</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="page__cards cards">
+      <div class="cards__container">
+        <div class="cards__card card" data-qa="card">
+          <img
+            class="card__img"
+            src="../src/images/imac.jpeg"
+            alt="producte image"
+          >
+          <h2 class="card__name-product">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </h2>
+          <p class="card__code-producte">
+            Код товара: 195434
+          </p>
+          <div class="card__rating">
+            <div class="card__stars">
+              <div class="card__star card__star_active"></div>
+              <div class="card__star card__star_active"></div>
+              <div class="card__star card__star_active"></div>
+              <div class="card__star card__star_active"></div>
+              <div class="card__star"></div>
+            </div>
```

Comment:
let's make stars a separate block
also, let's add active star background with css, not by hardcoding to each star card__star_active class